docs/helpers/Nightmare.md

```diff
@@ -384,18 +348,18 @@ Additional arguments can be passed as well, while `done` function is always last
 
 ```js
 let val = await I.executeAsyncScript(function(url, done) {
-  // in browser context
-  $.ajax(url, { success: (data) => done(data); }
+// in browser context
```

Comment:
All code examples in mustache fragments lost tabulation

Reply:
Yes, that's true. I don't how to fix that in this moment